docs/architecture/compaction/compaction-strategies.rst

```diff
@@ -70,7 +70,38 @@ Set the parameters for :ref:`Leveled Compaction <leveled-compaction-strategy-lcs
 Incremental Compaction Strategy (ICS)
 =====================================
 
-ICS is only available in ScyllaDB Enterprise. See the `ScyllaDB Enetrpise documentation <https://enterprise.docs.scylladb.com/stable/architecture/compaction/compaction-strategies.html>`_ for details.
+.. versionadded:: 2019.1.4 Scylla Enterprise
+
+.. note:: ICS is only available for Scylla Enterprise customers
+
+ICS principles of operation are similar to those of STCS, merely replacing the increasingly larger SSTables in each tier, by increasingly longer SSTable runs, modeled after LCS runs, but using larger fragment size of 1 GB, by default.
+
+Compaction is triggered when there are two or more runs of roughly the same size. These runs are incrementally compacted with each other, producing a new SSTable run, while incrementally releasing space as soon as each SSTable in the input run is processed and compacted. This method eliminates the high temporary space amplification problem of STCS by limiting the overhead to twice the (constant) fragment size, per shard.
+
+Incremental Compaction Strategy benefits
+----------------------------------------
+* Greatly reduces the temporary space amplification which is typical of STCS,  resulting in more disk space being available for storing user data.
+* The space requirement for a major compaction with ICS is almost non-existent given that the operation can release fragments at roughly same rate it produces new ones.
+
+If you look at the following screenshot the green line shows how disk usage behaves under ICS when major compaction is issued.
+
+.. image:: /architecture/compaction/screenshot.png
+
+Incremental Compaction Strategy disadvantages
+----------------------------------------------
+
+* Since ICS principles of operation are the same as STCS, its disadvantages are similar to STCS's, except for the temporary space amplification issue.
+
+Namely:
+
+* Continuously modifying existing rows results in each row being split across several SSTables, making reads slow, which doesn’t happen in Leveled compaction.
+* Obsolete data (overwritten or deleted columns) may accumulate across tiers, wasting space, for a long time, until it is finally merged. This can be mitigated by running major compaction from time to time.
```

Comment:
and by using Space Amplification Goal to encourage cross-tier compaction of the two largest tiers earlier, as data accumulates in the second-largest tier.

Reply:
yes, looks like we forgot to update this when SAG was introduced.